controllers/knative/knative.go

```diff
@@ -84,3 +102,100 @@ func GetKnativeAvailability(cfg *rest.Config) (*Availability, error) {
 		return result, nil
 	}
 }
+
+func GetWorkflowSink(ctx context.Context, c client.Client, workflow *operatorapi.SonataFlow, pl *operatorapi.SonataFlowPlatform) (*duckv1.Destination, error) {
+	if workflow == nil {
+		return nil, nil
+	}
+	if workflow.Spec.Sink != nil {
+		return workflow.Spec.Sink, nil
+	}
+	if pl != nil {
+		if pl.Spec.Eventing != nil {
```

Comment:
I think you can simplify this conditional with `pl != nil && pl.Spec.Eventing != nil`.

Reply:
Good idea. Updated.